packages/abi-coder/src/abi-coder.test.ts

```diff
@@ -1,491 +0,0 @@
-import { hexlify, concat } from '@ethersproject/bytes';
```

Comment:
All of the tests in this file are now part of the parameterized tests in `interface.test.ts`'s encoding testing part.